package.json

```diff
@@ -31,10 +31,14 @@
     "lint:ts": "tsc --noEmit",
     "prebuild": "rimraf dist",
     "build:copy-files": "babel-node ./scripts/copy-build-files.js",
-    "prepublishOnly": "yarn-or-npm build",
+    "prepublishOnly": "pinst --disable",
     "precommit": "lint-staged",
     "format": "prettier --write src/**/*.ts*",
-    "npm:publish": "np --contents=dist"
+    "npm:publish": "np --contents=dist",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook -o ./docs",
+    "postinstall": "husky install",
+    "postpublish": "pinst --enable"
```

Comment:
I don't think that we need `pinst` at all because the  `package.json` we publish is a light version of this one without any `scripts`.

Reply:
Makes sense ! I just followed husky docs blindly, should have looked more.